src/hotspot/share/oops/typeArrayKlass.hpp

```diff
@@ -73,6 +73,9 @@ class TypeArrayKlass : public ArrayKlass {
   // Copying
   void  copy_array(arrayOop s, int src_pos, arrayOop d, int dst_pos, int length, TRAPS);
 
+  // jvm support
```

Comment:
"jvm support" for what? I'm not even sure why we need `compute_modifier_flags` for array types when they have a fixed value. Can't we just hardwire them and only call `compute_modifier_flags` for instanceKlasses?

Reply:
ObjArrayKlass computes the modifier flags based on the bottom_klass, and there is a place that calls compute_modifier_flags in an assert that requires the virtual function, albeit for an assert, otherwise I would have made it not a virtual function.

Reply:
Okay I see now - the access modifier part of the modifiers for a reference array comes from the element type.

Reply:
But still "jvm support" in the comment really doesn't explain anything.